1-org/envs/shared/log_sinks.tf

```diff
@@ -72,6 +72,42 @@ module "bq_system_event_logs" {
   Audit Logs - Data Access
 *****************************************/
 
+resource "google_organization_iam_audit_config" "organization_data_access_config" {
+  count   = var.data_access_logs_enabled ? (var.parent_folder != "" ? 0 : 1) : 0
```

Comment:
cant we just do ` var.data_access_logs_enabled && var.parent_folder == ""  1 : 0 `
If I understand the logic correctly, if data_access_logs_enabled and if org i.e `parent_folder == "" ` then we need one `google_organization_iam_audit_config`

Reply:
You are right. Thanks for the suggestion.